kubernetes/samples/scripts/create-weblogic-domain/domain-home-on-pv/create-domain-inputs.yaml

```diff
@@ -162,6 +162,8 @@ createDomainFilesDir: wlst
 # istioEnabled specifies whether or not it is deployed under Istio.
 istioEnabled: false
 
-# Istio service mesh support is experimental.
 # istioReadinessPort specifies the WebLogic NAP port for use by the readiness probe
 istioReadinessPort: 8888
+
+# istioEnvoyPort specifies the WebLogic NAP port for use by the readiness probe
```

Comment:
Assuming our ping with Mark is as expected, delete this.

Reply:
removed